skbeam/core/accumulators/binned_statistic.py

```diff
@@ -158,9 +162,8 @@ def __init__(self, sample, statistic='mean',
             # Would be better to do this using bincount "weights", perhaps.
             thissample = sample[:, i]
             if mask is not None:
-                thissample[mask == 0] = (self.edges[i][0] -
-                                         0.01 * (
-                                             1 + np.fabs(self.edges[i][0])))
+                thissample[mask == 0] = np.floor(
```

Comment:
PS: a comment about this. This was a hack to make sure the masked values go out of range. We could consider a better alternative (but it would be nice to go through benchmarking before committing, and stash some of this somewhere).

Reply:
The floor makes certain it is below the threshold. There were some nasty corner cases where it didn't work otherwise. I can remove it otherwise.